includes/NormalizerFactory.php

```diff
@@ -0,0 +1,52 @@
+<?php
+
+namespace Dation\Woocommerce;
+
+use Symfony\Component\PropertyInfo\Extractor\PhpDocExtractor;
+use Symfony\Component\PropertyInfo\Extractor\ReflectionExtractor;
+use Symfony\Component\PropertyInfo\PropertyInfoExtractor;
```

Comment:
As you never seem to actually set `self::$normalizer` this statement is always true